### PR TITLE
fixed python not found error

### PR DESCRIPTION
**Pull request by miyamoto-san**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,13 +3,23 @@ FROM kthse/kth-nodejs:12.0.0
 WORKDIR /tmp/lms-export-to-ladok-2/
 # Copying only package.json to avoid reinstalling dependencies if only code has changed
 COPY package.json package-lock.json ./
+RUN apk update && \
+    apk add --no-cache --virtual .gyp \
+    python \
+    make \
+    g++
```
Comment (kaj) [inline Dockerfile:10]: Do we want to install those packages _after_ copying in the package.json files?  I would have assumed right after the `FROM` line?

File: Dockerfile [hunk 2]
```diff
@@ -3,13 +3,23 @@ FROM kthse/kth-nodejs:12.0.0
...
 RUN npm run build
 
 # Stage 1. Build the actual image
 FROM kthse/kth-nodejs:12.0.0
 WORKDIR /usr/src/app
+RUN apk update && \
+    apk add --no-cache --virtual .gyp \
+    python \
+    make \
+    g++
```
Comment (kaj) [inline Dockerfile:22]: Do we really need those development dependencies in the runtime image?

File: Dockerfile (hunk 1 above)
Comment (miyamoto-san) [reply Dockerfile:10]: Why do you suggest it matters?

File: Dockerfile (hunk 2 above)
Comment (miyamoto-san) [reply Dockerfile:22]: Yes, it fails otherwise.

File: Dockerfile (hunk 1 above)
Comment (kaj) [reply Dockerfile:10]: Not sure what defines that RUN as out-of-date or not, but maybe we don't have to run it for every change in package-lock.json?  On the other hand, maybe its better to get updates more often than not, and the cost of a pair of apk commands in the build is rather small.

File: Dockerfile (hunk 2 above)
Comment (kaj) [reply Dockerfile:22]: Hm.  I can imagine python being needed (or at least nice to have) runtime.  But make and g++?  That sounds like something that should have been done in the build step is done in the run step instead.  Or maybe just that some libc or libstdc++ that g++ depends on is what is really needed?

Doesn't really matter though, only downside of keeping it as is is that the build make take a few seconds more and the image gets slightly larger.

Comment (miyamoto-san) [reply Dockerfile:22]: You may right about that. Let me try to remove g++ and something else and see if it works! I think it's needed by pythong though and so on. But lets try before I say too much =)

File: Dockerfile (hunk 1 above)
Comment (miyamoto-san) [reply Dockerfile:10]: I moved it above the `COPY`. I am assuming you're implying it has something to do with caching?

File: Dockerfile (hunk 2 above)
Comment (miyamoto-san) [reply Dockerfile:22]: I have removed `g++` and will push in a sec!

**Pull request merged**